README.md

```diff
@@ -109,6 +109,20 @@ as `dw` to delete a word) work in multi-cursor mode. You have to
 manually set this because vim doesn't provide a way to see which keys *start*
 mappings; setting it to include motion commands like `j` can break things.
 
+### ```g:multi_cursor_paste_buffer_register``` (Defaut: `"`)
+The value of this option determines which register to use while
+saving/restoring paste buffer of the cursors. With this option you can select
+text in Visual mode, yank it, move the cursors, and paste it back. Each cursor
+will paste the text it yanked. `y` and `Y` are supported in Visual mode.
+Normal mode is supported but limited: `C`, `s`, `S`, `x`, `X`, `D` (commands
+without motions) work as usual. Only few of commands with motions are
+supported: `ci`, `ca`, `cb`, `ce`, `cw` and similar. `t`, `f`, `/` motions are
+not supported. Note that it is assumend that commands with motions such that
+`c` and `d` are added to `g:multi_cursor_normal_maps`.
+
+You should not change this option under normal circumstances. Setting value to
```

Comment:
Well, not anymore. The history behind this option is that I didn't immediately realize I could turn `clipboard` off while cursors are active. I personally have `clipboard+=unnamed,unnamedplus` and at first I was testing without that option and unnamed `@"` register worked. But when you turn that option on, looks like `@"` is not the cursor you want anymore, and then I started using `@+` instead. That's where all sorts of weird sh-t started to happen. I've seen behaviour I couldn't explain, literally: when the last statement in a function is `let ...`, the state gets screwed up during the call, but when I add `echom ...`, it works just fine. Anyway, after I discovered that trick to save/restore clipboard that option became redundant, I'll remove it.